### DISCUSSION TITLE
Keeping data added while importing large dataset from pocket

### DISCUSSION THREAD

**Discussion by GreenBlast** [go-shiori/shiori#848] [Feature requests] [answered]
Hi. 

Thanks all for the project. Looks amazing.

I want to import my pocket current + archived (read) articles (Around 20k over the span of 15 years). 
I would really like to retain the timestamp of creation in pocket, as the date goes way back, and the time tracking is meaningful.

But using the pocket2shiori.sh script the article is created with a new timestamp (Makes sense)

I was thinking that maybe if it was possible to add a flag to override the timestamp that gets a timestamp as a parameter so that the creation date will be stored as the timestamp.

**Comment by Monirzadeh** [comment, marked as answer]
import functionality need refactor but for now maybe you can edit that in database manually.
i am not sure it compliantly work or not but you can test that and automatic that with a script and see the result.
bookmark table modified section
before done that in current setup test that on a sample data in a separate shiori instance to be sure it is working correctly.
always backup data before manipulate that.

**Reply by GreenBlast** [reply]
Actually a great idea, I think I can probably import everything and then hack something that will change the timestamp in the db. Didn't thought about that.

**Reply by Monirzadeh** [reply]
please backup before modify db and if you want share your script with others in documentation.
